app/services/AdjacentCellsCalculator.coffee

```diff
@@ -0,0 +1,24 @@
+Tuple = require "../models/Tuple"
+
+class AdjacentCellsCalculator
+
+  # Param grid is a GameGrid
+  constructor: (@grid, @cells, @x, @y) ->
+
+  # Gets the adjacent cells
+  calculate: () => 
+    for i in [@y - 1, @y, @y + 1]
+      for j in [@x - 1, @x, @x + 1]
+        continue if i is @y and j is @x
```

Comment:
I think `continue` skips to the next iteration of the loop. So here I'm trying to skip excecution for the current cell. Unless `continue` does something else?

<div>-------- Original message --------</div><div>From: Shubham Gogna <notifications@github.com> </div><div>Date:07/20/2015  9:07 AM  (GMT-08:00) </div><div>To: sf-interns/mathswipe <mathswipe@noreply.github.com> </div><div>Cc: Eric Campbell <eric.campbell@originate.com> </div><div>Subject: Re: [mathswipe] EC Adjacent Vertices Calculator (#8) </div><div>
</div>In app/services/AdjacentCellsCalculator.coffee:

> @@ -0,0 +1,24 @@
> +Tuple = require "../models/Tuple"
> +
> +class AdjacentCellsCalculator
> +
> +  # Param grid is a GameGrid
> +  constructor: (@grid, @cells, @x, @y) ->
> +
> +  # Gets the adjacent cells
> +  calculate: () => 
> +    for i in [@y - 1, @y, @y + 1]
> +      for j in [@x - 1, @x, @x + 1]
> +        continue if i is @y and j is @x
unless i is @y and j is @x

—
Reply to this email directly or view it on GitHub.